app/templates/index.hbs

```diff
@@ -1,44 +1,115 @@
-                    </c.header>
-                    <c.content>
-                        <p>
-                            Lorem ipsum dolor sit amet
-                        </p>
-                    </c.content>
-                    <c.footer>
-                        <LinkTo class="au-c-button" @route="administrative-units">
-                            Ga naar databank
-                        </LinkTo>
-                    </c.footer>
-                </AuCard>
-            </li>
-            <li class="au-o-grid__item au-u-1-2@medium au-u-1-3@large">
-                <AuCard @flex={{true}} as |c|>
-                    <c.header @badgeSkin="brand" @badgeIcon="alert-triangle">
-                        <AuHeading @level="2" @skin="4">
-                            Personen
-                        </AuHeading>
-                    </c.header>
-                    <c.content>
-                        <p>
-                            Lorem ipsum dolor sit amet
-                        </p>
-                    </c.content>
-                    <c.footer>
-                        <LinkTo class="au-c-button" @route="people">
-                            Ga naar databank
-                        </LinkTo>
-                    </c.footer>
-                </AuCard>
-            </li>
+<div class="au-c-main-container">
+  <div class="au-c-main-container__sidebar">
+    <div class="au-d-component-block au-c-sidebar">
+      <div class="au-c-sidebar__content">
+        <div class="au-c-sidebar__header">
+          <AuHeading @level="6" @skin="6">Menu modules</AuHeading>
+        </div>
+        <ul class="au-c-list-navigation">
+          <li class="au-c-list-navigation__item">
+            <AuNavigationLink @linkRoute="administrative-units">
+              Bestuurseenheden
+            </AuNavigationLink>
+          </li>
+          <li class="au-c-list-navigation__item">
+            <AuNavigationLink @linkRoute="people">
+              Personen
+            </AuNavigationLink>
+          </li>
         </ul>
+      </div>
+      <div class="au-c-sidebar__footer">
+        <a class="au-c-link au-c-link--secondary au-c-link--icon" href="" target="_blank" rel="noopener noreferrer">
+          <AuIcon @icon="manual" @alignment="left" />
+          Handleiding
+        </a>
+      </div>
     </div>
+  </div>
+  <div class="au-c-main-container__content au-c-main-container__content--block">
+    <div
+      class="au-o-box au-o-flow au-o-flow--large"
+    >
+      <AuHeading @skin="2">
+        Lokale Financiën - Jan Janssens
+      </AuHeading>
+
+      <AuHeading @level="2" @skin="3" class="au-u-margin-bottom">
+        Beschikbare modules
+      </AuHeading>
+
+      <ul class="au-o-grid au-o-grid--small">
+        <li class="au-o-grid__item au-u-1-2@medium au-u-1-3@large">
+          <AuCard @flex={{true}} as |c|>
+            <c.header @badgeSkin="brand" @badgeIcon="sitemap">
+              <AuHeading @level="2" @skin="4">
```

Comment:
I think these need to be level 3 since there is already a h2 parent, right?

Reply:
Same result.

Reply:
Not sure what you mean with that :smile:

Reply:
Ah yea, the `@level` only changes the element, not the styling.